modules/exploits/multi/http/nuuo_nvrmini_upgrade_rce.rb

```diff
@@ -0,0 +1,76 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'        => 'NUUO NVRmini - upgrade_handle.php Remote Command Execution',
+      'Description' => %q{
+        NUUO NVRmini  ip camera web application. The upgrade_handle.php file is affected by the remote command execution vulnerability.
+      },
+      'License' => MSF_LICENSE,
+      'Author'  =>
+        [
+          'Berk Dusunur <@berkdusunur>', 
```

Comment:
The whitespace at the end of line was making `msftidy` unhappy.

```suggestion
          'Berk Dusunur <@berkdusunur>',
```

```
$ ./.git/hooks/post-merge
[*] Running msftidy.rb in ./.git/hooks/post-merge mode
--- Checking new and changed module syntax with tools/dev/msftidy.rb ---
modules/exploits/multi/http/nuuo_nvrmini_upgrade_rce.rb:20 - [WARNING] Spaces at EOL
```

Reply:
solved